qa-jcoin.planx-pla.net/portal/gitops.json

```diff
@@ -82,25 +87,14 @@
           "icon": "files",
           "link": "/files",
           "color": "#a2a2a2",
-          "name": "Files"
+          "name": "Datasets"
         },
         {
           "icon": "query",
           "link": "/query",
           "color": "#a2a2a2",
           "name": "Query"
         },
-        {
-          "icon": "dictionary",
-          "link": "https://qa-jcoin.planx-pla.net/dashboard/Secure/index.html",
-          "color": "#a2a2a2",
-          "name": "HUB Documentation"
-        },
-        {
-          "name": "Notebook Browser",
-          "link": "/resource-browser",
```

Comment:
removing the link to the MOUD notebook?

Reply:
@paulineribeyre  Does this sound ok?

Reply:
ok

Reply:
@mbkranz should we hold off merging until Phil has a look?